### ISSUE TITLE
Cannot run the quickstarts/java/camel-cdi with the camel maven plugin

### ISSUE THREAD

**Issue by christian-posta** [fabric8io/fabric8#4220]
> mvn camel:run 

doesn't work, get this odd dependency trace:

http://pastebin.com/SEU7tvZc

It also follows that any simple java cdi project created from the corresponding archetype doesn't allow camel:run to work either.

> mvn exec:java

does work, so using that as a workaround

**Comment by davsclaus** [comment]
yeah i noticed that too in the plain apache camel cdi example and archetype.

Its a Camel problem and not fabric8.

**Comment by davsclaus** [comment]
Logged a ticket to not forget
https://issues.apache.org/jira/browse/CAMEL-8884

**Comment by davsclaus** [comment]
Fixed in Camel 2.16 onwards. We will upgrade to it when its released
